ui/pages/confirmations/confirm-transaction/confirm-token-transaction-switch.js

```diff
@@ -179,7 +183,10 @@ export default function ConfirmTokenTransactionSwitch({ transaction }) {
             transaction={transaction}
             ethTransactionTotal={ethTransactionTotal}
             fiatTransactionTotal={fiatTransactionTotal}
-            hexMaximumTransactionFee={hexMaximumTransactionFee}
+            hexMaximumTransactionFee={sumHexes(
+              hexMaximumTransactionFee,
+              layer1GasFee ?? 0x0,
```

Comment:
Minor, but should this fallback be a string since it's adding hex strings, or will it cast to zero?